packages/libs/append-vec/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+# Please also update the badges in `README.md`, `src/lib.rs`, and `macros/` when changing this
+channel = "nightly-2022-09-27"
+components = ['rust-src', 'clippy', 'rustfmt']
```

Comment:
For what we are going to use `rust-src`? I think as long as we don't have UI tests we don't need it.

Reply:
true, this needs to be replaced with `miri`